miro_preprocessor/main.tf

```diff
@@ -0,0 +1,44 @@
+module "xml_to_json_converter" {
+  source        = "../terraform/ecs_script_task"
+  task_name     = "xml_to_json_converter"
+  app_uri       = "${module.ecr_repository_xml_to_json_converter.repository_url}:${var.release_ids["xml_to_json_converter"]}"
+  task_role_arn = "${module.ecs_xml_to_json_converter_iam.task_role_arn}"
+
+  # This script has to load the XML files into memory, so make sure it
+  # has plenty of overhead.
+  memory = 2000
```

Comment:
If we’re doing stream parsing, is that true any more?

Reply:
In order to write the whole JSON file at some point _that_ needs to be accumulated in memory - so while the stream parsing will help I think the memory requirements will be similar.